src/wmcads/components/page-header/_page-header.scss

```diff
@@ -1,108 +1,108 @@
-.wmcads-page-header {
```

Comment:
Accessibility fix - maintain layout without text overlap when using text-only zoom in browser